manifests/examples/virtualbox/stretch-vagrant.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: debian-{system.release}-{system.architecture}-{%y}{%m}{%d}
+provider:
+  name: virtualbox
+  guest_additions: /usr/share/virtualbox/VBoxGuestAdditions.iso
+bootstrapper:
+  workspace: /target/stretch
```

Comment:
Sorry to say, but your specific work process is not enough of a reason to justify a change like this :-/

However, you should be able to inline the `run_id` using manifest vars, so that your images are uniquely named.